code/datums/mind.dm

```diff
@@ -1718,8 +1718,7 @@
 			if(istype(S, type))
 				continue
 		S.charge_counter = delay
-		spawn(0)
-			S.start_recharge()
+		addtimer(S, "start_recharge", 0)

```

Comment:
I made a pr to make addtimer(..,..,0) basically equate to a set waitfor = 0 call of the proc.